Update Snail people for reasons beyond comprehension

Debonaircorsiar opened the pull request:
Balances out snail people, giving them a speed(from 6 to 2, the same as golems) and unarmed damage boost(from 0.5 to 5 brute a hit) while lowering their natural armor and burn weakness, effectively making them playable.

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Changes snails to not be shit.

## Why It's Good For The Game

Because its something I said I was gonna do


## Changelog

Changed:
Burnmod from 2 to 1.5
Speedmod from 6 to 2 
Punchdamagehigh from 0.5 to 5
Revalued the shell backpacks armor so grey tiders weren't walking around in armored vests round start, but still give a a small improvement to deal with the speed and burn damage traits.

Edit: 

Removed their snailcrawl; When snails laid down, they would leave lubed up tiles behind them and get a speed boost
Removed their phobia of salt, making them inable to move for any reason over salt piles.
Removed their noslip trait

Commit:
Update Snail people for reasons beyond comprehension

Balances out snail people, giving them a speed(from 6 to 2, the same as golems) and unarmed damage boost(from 0.5 to 5 brute a hit) while lowering their natural armor and burn weakness, effectively making them playable.

qwertyquerty commented:
Please write the changelog out

Commit:
Update snail.dm

ghost commented:
Okay I think I did it right?

qwertyquerty commented:
Looks like you made this race kind of OP.

ghost commented:
The armor is near negligible unless you're wearing additional armor, which even then won't boost it my a significant amount, the punch damage from 0.5 to 5 is more then reasonable given they literally would deal less then 1 brute a hit, the speed I was debating on lowering to a 3 until I relized how unbearable it would be for people to wear hardsuits with this race,.

All in all their just humans who take more burn damage, can't eat salt, walk slower, and when wearing armor are slightly more robust in defence.

Commit:
Update snail.dm

ghost commented:
That one there is to remove snailcrawl for now, and to remove snails having no slip because I forgot to do that

Commit:
Salt fix

Snails are literally unable to move over salt tiles whatsoever, whether it is willingly or forcibly, this should fix that.

ghost commented:
That one should fix their salt tile bullshit shinanigains

CthulhuOnIce commented:
travis failed

CthulhuOnIce commented:
```
code/modules/mob/living/carbon/human/species_types/snail.dm:36:error: incorrect number of macro arguments
code/modules/mob/living/carbon/human/species_types/snail.dm:42:error: incorrect number of macro arguments
```

CthulhuOnIce commented on code/modules/mob/living/carbon/human/species_types/snail.dm:
```diff
@@ -33,14 +33,13 @@
 			C.equip_to_slot_or_del(new /obj/item/storage/backpack/snail(C), SLOT_BACK)
-	C.AddComponent(/datum/component/snailcrawl)
-	ADD_TRAIT(C, TRAIT_NOSLIPALL, SPECIES_TRAIT)
+	ADD_TRAIT(C, SPECIES_TRAIT)
```
revert this

CthulhuOnIce commented on code/modules/mob/living/carbon/human/species_types/snail.dm:
```diff
@@ -33,14 +33,13 @@
 	var/datum/component/CP = C.GetComponent(/datum/component/snailcrawl)
 	CP.RemoveComponent()
-	REMOVE_TRAIT(C, TRAIT_NOSLIPALL, SPECIES_TRAIT)
+	REMOVE_TRAIT(C, SPECIES_TRAIT)
```
Revert this

CthulhuOnIce reviewed (changes requested):
revert some changes

Commit:
Update snail.dm

Commit:
Update food.dm
